layers/+emacs/org/README.org

```diff
@@ -423,6 +424,18 @@ More information about org-roam package (including manual) can be found at [[htt
 To install support for [[https://github.com/org-roam/org-roam-server][org-roam-server]] set the variable =org-enable-roam-server=
 to =t=.
 
+*** Org-roam-protocol support
+To install support for [[https://www.orgroam.com/manual.html#Roam-Protocol][Org Roam Protocol]] set the variable
+=org-enable-roam-protocol= to =t=.
+
+#+begin_src emacs-lisp
+  (setq-default dotspacemacs-configuration-layers '(
+    (org :variables
+         org-enable-roam-protocol t)))
+#+end_src
+
+And create a desktop application as described in the [[https://www.orgroam.com/manual.html#Roam-Protocol][org-roam manual]].
```

Comment:
According to Desktop Entry Specification of Freedesktop, it's either Desktop Entry File or Desktop File. 

*desktop application* is a misnomer.

Reply:
Noted, I used the terminoligy of the org-roam manual, but if the spec says something different then that's definitively  the terminology that should be used.